axelrod/strategies/titfortat.py

```diff
@@ -194,7 +232,12 @@ def strategy(opponent):
 
 class HardTitFor2Tats(Player):
     """A variant of Tit For Two Tats that uses a longer history for
-    retaliation."""
+    retaliation.
+
+    Names:
+
+    - Hard Tit For Two Tats:
```

Comment:
Cool. In those cases can you put:

```
     - Hard Tit For Two Tats: Reference required.
```

It's still better to have the name clearly there for the case of people looking for it.

Although I think you can probably find Hard Tit For Two Tats in various papers.